packages/flutter/lib/src/material/stepper.dart

```diff
@@ -195,6 +195,8 @@ class Stepper extends StatefulWidget {
   ///
   /// This callback which takes in a context and two functions: [onStepContinue]
   /// and [onStepCancel]. These can be used to control the stepper.
+  /// For example, keeping track of the [currentStep] within the callback can
+  /// change the text of the button, like 'save' at the last step.
```

Comment:
What do you mean by 'save' here?

Reply:
Ah, it's supposing that users are filling some forms in `Stepper`.
```suggestion
  /// change the text of the continue or cancel button as per which step are users at.
```
How about updating like this? Would it be better?

Reply:
That sounds good to me! Thank you!